site/app/templates/polls/ViewPoll.twig

```diff
@@ -1,6 +1,22 @@
 <div class="content">
     <h1>
-        {{ poll.getName() }}
+        {{ poll.getEndDate() | date("F jS, Y \\a\\t g:i A")}} 
+        {% set duration = poll.getDuration() %}
+        {% set durationString = '' %}
+        {% if duration.days > 0 %}
+            {% set durationString = durationString ~ duration.days ~ ' day(s) ' %}
+        {% endif %}
+        {% if duration.h > 0 %}
+            {% set durationString = durationString ~ duration.h ~ ' hour(s) ' %}
+        {% endif %}
+        {% if duration.i > 0 %}
+            {% set durationString = durationString ~ duration.i ~ ' minute(s) ' %}
+        {% endif %}
+        {% if duration.s > 0 %}
+            {% set durationString = durationString ~ duration.s ~ ' second(s) ' %}
+        {% endif %}
```

Comment:
You can make the UI ever so slightly by handling the singular and plural cases automatically, rather than adding `(s)` to the end of every string.

Reply:
This is just to test the duration and time. It should be the Poll Name and a JS timer displayed.